### DISCUSSION TITLE
Any one updated to MacOS Sequoia and runnin Skip fine?

### DISCUSSION THREAD

**Discussion by davidakoontz** [skiptools/skip#243] [General]
I’m about to update to latest MacOS. 
wondered how Skip tool handle this update?
Experience anyone?

**Comment by marcprux** [comment]
I have updated to macOS 15 Sequoia and Xcode 16, and Skip is running fine for me.

**Reply by davidakoontz** [reply]
Just what I hoped!!

**Reply by davidakoontz** [reply]
Running Great on MacOS 15 Sequoia & Xcode 16.  No problems!!

**Reply by danielcr12** [reply]
How was the performance of your project when I create a new app from the template and I built to install the Android phone I feel like the app is running at low FPS or without any optimizations

**Reply by davidakoontz** [reply]
I have little to compare with - I'm not an Android user.  And quit working this project some months ago.
